client/signup/jetpack-connect/new-site/style.scss

```diff
@@ -0,0 +1,223 @@
+// Colors
+$green-jetpack:     #8cc258; // RGB 140, 194, 88
+
+// Structure
+.jetpack-new-site {
+	max-width: 960px;
+	margin: 35px auto 0;
+	text-align: center;
+
+	/* @include breakpoint( ">480px" ) {
+		margin-bottom: 16px;
+		padding: 24px;
+	} */
+}
+
+.jetpack-new-site__header {
+	margin-bottom: 3em;
+
+	@include breakpoint( "<660px" ) {
+		margin: 0 2em 2em;
+	}
+}
+
+.jetpack-new-site__header-title {
+	margin-bottom: .35em;
+	font-size: 34px;
+	font-weight: 300;
+	color: $gray-dark;
+}
+
+.jetpack-new-site__header-text {
+	font-size: 16px;
+	color: $gray;
+}
+
+.jetpack-new-site__content {
+	display: flex;
+	align-items: stretch;
+	justify-content: center;
+	padding-bottom: 2em;
+}
+
+.jetpack-new-site__wpcom-site,
+.jetpack-new-site__jetpack-site {
+	z-index: 1;
+	display: flex;
+	flex-direction: column;
+	width: 360px;
+	margin: 0px;
+	transition: transform .25s ease, box-shadow .35s ease, z-index .15s ease;
+
+	&:hover {
+		z-index: 3;
+		transform: scale(1.075);
+		box-shadow: 0 0 0 1px transparentize( lighten( $gray, 20% ), .5 ),
+		0 1px 2px lighten( $gray, 30% ), 0 7px 18px transparentize( lighten( $gray, 15% ), .5 );
+		transition: transform .25s ease, box-shadow .35s ease, z-index .175s ease;
+	}
+
+	@include breakpoint( "<660px" ) {
+		display: none;
+	}
+}
+
+.jetpack-new-site__wordpress-logo,
+.jetpack-new-site__jetpack-logo {
+	flex: 1;
+	max-width: 72px;
+	max-height: 72px;
+	margin: 1.5em auto;
+	transition: .25s transform ease;
+}
+
+.jetpack-new-site__wordpress-logo {
+	fill: $blue-wordpress;
+}
+
+.jetpack-new-site__jetpack-logo {
+	fill: $green-jetpack;
+}
+
+.jetpack-new-site__card-title {
+	margin-bottom: .75em;
+	font-size: 24px;
+	line-height: 1.25em;
+}
+
+.jetpack-new-site__card-description {
+	margin-bottom: 1em;
+	color: lighten( $gray-dark, 25% );
+}
+
+.jetpack-new-site__button-holder {
+	display: flex;
+	flex: 1;
+	align-self: flex-end;
+	width: 100%;
+}
+
+.jetpack-new-site__wpcom-site {
+	
+	.button {
+		align-self: flex-end;
+		width: 100%;
+		margin-top: 16px;
+	}
+}
+
+.jetpack-new-site__jetpack-site,
+.jetpack-new-site__mobile-jetpack-site {
+
+	.jetpack-connect__connect-button-card {
+		background: transparent;
+	}
+
+	.jetpack-connect__site-address-container  {
+		position: relative;
+
+		.gridicon {
+			position: absolute;
+			top: 8px;
+			left: 8px;
+			color: lighten( $gray, 20% );
+		}
+
+		.form-text-input {
+			padding-left: 40px;
+		}
+
+		.spinner {
+			position: absolute;
+			right: 8px;
+			top: 10px;
+		}
+	}
+
+	label {
+		display: none;
+	}
+
+	input[type=text] {
+		&:focus {
+			border-color: $green-jetpack;
+			box-shadow: 0 0 0 2px lighten( $green-jetpack, 25% );
+		}
+	}
+
+	.button {
```

Comment:
Is this a style we'd want to reuse anywhere else? Because it may make sense to move it to a prop of `Button`.